Code/PgSQL/rdkit/update_sql/rdkit--4.1.1--4.2.sql.in

```diff
@@ -0,0 +1,9 @@
+CREATE OR REPLACE FUNCTION mol_from_smiles(text) 
+RETURNS mol 
+@RDKIT_PARALLEL_SAFE@
+AS 'select mol_from_smiles($1::cstring);' 
+LANGUAGE SQL IMMUTABLE 
+RETURNS NULL ON NULL INPUT;
+
+CREATE CAST (text as mol) WITH FUNCTION mol_from_smiles(text) AS IMPLICIT;
```

Comment:
@greglandrum FYI I think these implicit casts cause problems if the rdkit extension is not loaded in the public schema (postgres can't find them)